### DISCUSSION TITLE
使用 TableExport 导出数据时, 怎样添加统计行

### DISCUSSION THREAD

**Discussion by Vincent-0805** [dotnetcore/BootstrapBlazor#3170] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

使用 TableExport 导出数据时, 怎样添加统计行
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/142959771/1af4cd15-99b9-4bbd-97b1-773891b3a893)
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/142959771/345fd8b9-ef28-4d3b-b6f1-e7b854869ad5)

**Comment by bb-auto** [comment]
@Vincent8725 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment, marked as answer]
@Vincent8725 导出工具一般都可以做这种汇总处理的，就是 Excel 中的公式，在导出工具的功能中实现

请点击👇 `Make as answer`
